src/CampaignAdmin.php

```diff
@@ -396,6 +406,42 @@ public function readCampaign(HTTPRequest $request)
     }
 
     /**
+     * REST endpoint to delete a campaign item.
+     *
+     * @param HTTPRequest $request
+     *
+     * @return HTTPResponse
+     */
+    public function removeCampaignItem(HTTPRequest $request)
+    {
+        // Check security ID
+        if (!SecurityToken::inst()->checkRequest($request)) {
+            return new HTTPResponse(null, 400);
+        }
+
+        $campaignID = $request->param('CampaignID');
+        $itemID = $request->param('ItemID');
+
+        if (!$campaignID ||
+            !is_numeric($campaignID) ||
+            !$itemID ||
+            !is_numeric($itemID)) {
```

Comment:
having `!$campaignID` and `!$itemID` seems redundant, if an invalid numeric ID is provided (`0`) it should return a `404` error